loader/src/main/java/net/neoforged/fml/loading/progress/StartupNotificationManager.java

```diff
@@ -31,22 +31,22 @@ public static List<ProgressMeter> getCurrentProgress() {
     public static ProgressMeter prependProgressBar(final String barName, final int count) {
         var pm = new ProgressMeter(barName, count, 0, new Message(barName, Message.MessageType.ML));
         synchronized (progressMeters) {
-            progressMeters.addLast(pm);
+            progressMeters.addFirst(pm);
         }
         return pm;
     }
 
     public static ProgressMeter addProgressBar(final String barName, final int count) {
         var pm = new ProgressMeter(barName, count, 0, new Message(barName, Message.MessageType.ML));
         synchronized (progressMeters) {
-            progressMeters.push(pm);
+            progressMeters.addLast(pm);
         }
         return pm;
     }
 
     public static void popBar(final ProgressMeter progressMeter) {
         synchronized (progressMeters) {
-            progressMeters.remove(progressMeter);
+            progressMeters.removeLastOccurrence(progressMeter);
         }
```

Comment:
Why were these reordered?  Were they incorrect previously?

Reply:
Yes. This makes new progress bars render below old ones. (Previously a new one would render higher than an old one)

Reply:
> This makes new progress bars render below old ones. (Previously a new one would render higher than an old one)

Isn't that what we want? For instance, if you had a main progress bar for each registry, and then another one showing progress through the mod list, I would argue the mod one should be shown more prominently since it moves faster and has more useful information.

Reply:
As discussed on Discord we want the outer step to be above the inner step.